.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**IMPORTANT: please make sure you ask yourself all intro questions and fill all sections of the template.**
+
+**Before we start...:**
+
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+
+**Version:**
+
+Inform what version of Dash you are using.
```

Comment:
We are not versioning dash, should be start versioning it or change this to what branch/commit is being used?

Reply:
Good point. Template adjusted :)